### PR TITLE
Exploding water tanks now actually throw the water out

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Actually all tanks do, which means you can shoot the beer keg for a beer explosion.

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Thanks @nullbear

**Comment by Riktheprick** [comment]
Yes please.

**Comment by Kromgar** [comment]
Yeah so make buildable tanks we can fill with chems and can link together to mix chems rapidly.

1000u of napalm suddenly explodes out

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -49,27 +49,20 @@
 
 
 /obj/structure/reagent_dispensers/proc/boom()
+	visible_message("<span class='danger'>\The [src] ruptures!</span>"
```
Comment (PKPenguin321) [inline code/modules/reagents/reagent_dispenser.dm]: )

**Comment by Dorsisdwarf** [comment]
Wait, does this give people lethal alcohol poisoning if they get sprayed with 1000u of beer?

**Comment by GunHog** [comment]
>Wait, does this give people lethal alcohol poisoning if they get sprayed with 1000u of beer?
### 100% MAGNI BRONZEBEARD APPROVED

**Comment by nullbear** [comment]
Dorsidwarf, the way chemsplash code works is that it divides the units by the turfs it splashes to.

In your typical explosion, of 1000units, it'll apply about 7.5 units per mob. If you're in a smaller room, it'll apply more. If you're in a 1x1 tile on top of the keg, you might be able to get the full 1000 units.

**Comment by KorPhaeron** [comment]
I don't think alcohol applies via touch anyway

**Comment by nullbear** [comment]
Well then it won't do anything :p

**Comment by Dorsisdwarf** [comment]
ooh, how does this work with High-Pressure tanks?

**Comment by nullbear** [comment]
high-pressure tanks are being depreciated, and are a different type of object anyways @Dorsisdwarf

**Pull request merged**